packages/core/package.json

```diff
@@ -5,29 +5,19 @@
   "types": "dist/lib/index.d.ts",
   "description": "stitch",
   "dependencies": {
+    "bson": "^2.0.7",
+    "@types/bson": "^1.0.8",
```

Comment:
[minor] I don't think the `@types` should be dependencies of core (especially the jest and ones). This would seem to mean that all projects that include the `stitch-web` module would have to bring in these dependencies, even if they don't have Typescript.

If anything, these should just be made `devDependencies` of the module above that is trying to use them.